src/utilstrencodings.h

```diff
@@ -71,6 +71,21 @@ constexpr bool IsDigit(char c)
     return c >= '0' && c <= '9';
 }
 
+/**
+ * Tests if the given character is a whitespace character. The whitespace characters
+ * are: space, form-feed ('\f'), newline ('\n'), carriage return ('\r'), horizontal
+ * tab ('\t'), and vertical tab ('\v').
+ *
+ * This function is locale independent. Under the C locale this function gives the
+ * same result as std::isspace.
```

Comment:
nit: Could be more clear: ", giving the same result as std::isspace does under the C locale."